### PR TITLE
XSIAM_PB_User_added_to_local_administrator_group

### PR DESCRIPTION
**Pull request by OmriItzhak**
## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Related Issues
fixes: https://jira-dc.paloaltonetworks.com/browse/CIAC-11378

## Description
A new core playbook and trigger for XSIAM designed to handle the alert:
User added to local administrator group using a PowerShell command

## Must have
- [ ] Tests
- [ ] Documentation

### CODE REVIEW

File: Packs/Core/Playbooks/playbook-User_added_to_local_administrator_group_using_a_PowerShell_command.yml [hunk 1]
```diff
@@ -0,0 +1,639 @@
...
+    isautoswitchedtoquietmode: false
+  "76":
+    id: "76"
+    taskid: 43d9acfc-9cd7-43f6-8675-484582c3ac4d
+    type: regular
+    task:
+      id: 43d9acfc-9cd7-43f6-8675-484582c3ac4d
+      version: -1
+      name: Terminate the causality process
+      description: Terminate a process tree by its causality ID. Available only for XSIAM 2.4.
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-User_added_to_local_administrator_group_using_a_PowerShell_command.yml:375]:
```suggestion
      description: Terminate a process tree by its causality ID. Available only for Cortex XSIAM 2.4.
```

File: Packs/Core/Playbooks/playbook-User_added_to_local_administrator_group_using_a_PowerShell_command.yml [hunk 2]
```diff
@@ -0,0 +1,639 @@
...
+    isautoswitchedtoquietmode: false
+  "80":
+    id: "80"
+    taskid: 7d039298-b7e0-44c1-8f77-39e71f387d96
+    type: condition
+    task:
+      id: 7d039298-b7e0-44c1-8f77-39e71f387d96
+      version: -1
+      name: Analyst approval to remove user from local Administrator group
+      description: Analyst approval is required to remove the user from local Administrator group.
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-User_added_to_local_administrator_group_using_a_PowerShell_command.yml:526]:
```suggestion
      description: Analyst approval is required to remove the user from the local Administrator group.
```

File: Packs/Core/Playbooks/playbook-User_added_to_local_administrator_group_using_a_PowerShell_command.yml [hunk 3]
```diff
@@ -0,0 +1,639 @@
...
+  }
+inputs: []
+inputSections:
+- inputs: []
+  name: General (Inputs group)
+  description: Generic group for inputs
+outputSections:
+- outputs: []
+  name: General (Outputs group)
+  description: Generic group for outputs
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-User_added_to_local_administrator_group_using_a_PowerShell_command.yml:635]:
```suggestion
  description: Generic group for inputs.
outputSections:
- outputs: []
  name: General (Outputs group)
  description: Generic group for outputs.
```

File: Packs/Core/Playbooks/playbook-User_added_to_local_administrator_group_using_a_PowerShell_command_README.md [hunk 4]
```diff
@@ -0,0 +1,52 @@
+This playbook is designed to handle the alert
+'User added to local administrator group using a PowerShell command'
+
+The playbook executes the following stages:
+
+Investigation:
+Check the following parameters to determine if remediation actions are needed:
+- XDR alerts related to the hostname by MITRE tactics indicating malicious activity.
+- Whether the process is signed.
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-User_added_to_local_administrator_group_using_a_PowerShell_command_README.md:9]:
```suggestion
- Whether the process is unsigned.
```

File: Packs/Core/Playbooks/playbook-User_added_to_local_administrator_group_using_a_PowerShell_command_README.md [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+Remediation:
+Handles malicious alerts by terminating the relevant processes and removing the user from the local Administrators group.
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-User_added_to_local_administrator_group_using_a_PowerShell_command_README.md:12]:
```suggestion
Handles malicious alerts by terminating the relevant processes and requesting the analyst's approval to remove the user from the local Administrators group.
```

File: Packs/Core/Playbooks/playbook-User_added_to_local_administrator_group_using_a_PowerShell_command_README.md [hunk 6]
```diff
@@ -0,0 +1,52 @@
...
+
+This playbook uses the following sub-playbooks, integrations, and scripts.
+
+### Sub-playbooks
+
+This playbook does not use any sub-playbooks.
+
+### Integrations
+
+* CortexCoreIR
```
Comment (ShirleyDenkberg) [inline Packs/Core/Playbooks/playbook-User_added_to_local_administrator_group_using_a_PowerShell_command_README.md:25]:
```suggestion
CortexCoreIR
```

File: Packs/Core/ReleaseNotes/3_0_71.md [hunk 7]
```diff
@@ -0,0 +1,24 @@
...
+
+Remediation:
+Handles malicious alerts by terminating the relevant processes and removing the user from the local Administrators group.
+Handles non-malicious alerts identified during the investigation.
+
+#### Triggers Recommendations
+
+##### New: User added to local administrator group using a PowerShell command
+
+- New: This trigger is responsible for handling 'User added to local administrator group using a PowerShell command' alert.
```
Comment (ShirleyDenkberg) [inline Packs/Core/ReleaseNotes/3_0_71.md:24]:
```suggestion
New: This trigger is responsible for handling 'User added to local administrator group using a PowerShell command' alert.
```

**Comment by ShirleyDenkberg** [comment]
@ssokolovich Doc review completed.

File: Packs/Core/Playbooks/playbook-User_added_to_local_administrator_group_using_a_PowerShell_command.yml [hunk 8]
```diff
@@ -0,0 +1,724 @@
...
+  name: General (Inputs group)
+  description: Generic group for inputs.
+outputSections:
+- outputs: []
+  name: General (Outputs group)
+  description: Generic group for outputs.
+outputs: []
+tests:
+- No tests (auto formatted)
+fromversion: 6.10.0
```
Comment (ssokolovich) [inline Packs/Core/Playbooks/playbook-User_added_to_local_administrator_group_using_a_PowerShell_command.yml:724]:
```suggestion
fromversion: 8.8.0
```

File: Packs/Core/Playbooks/playbook-User_added_to_local_administrator_group_using_a_PowerShell_command.yml [hunk 9]
```diff
@@ -0,0 +1,725 @@
+id: User added to local administrator group using a PowerShell command
+version: -1
+name: User added to local administrator group using a PowerShell command
+description: |-
+  This playbook is designed to handle the alert
+  'User added to local administrator group using a PowerShell command'
```
Comment (umishkin) [inline Packs/Core/Playbooks/playbook-User_added_to_local_administrator_group_using_a_PowerShell_command.yml:6]: you put here ' ' but in your first playbook you didn't put quotes
let's keep consistent without quotes

File: Packs/Core/Playbooks/playbook-User_added_to_local_administrator_group_using_a_PowerShell_command.yml [hunk 10]
```diff
@@ -0,0 +1,725 @@
...
+
+  The playbook executes the following stages:
+
+  Investigation:
+  Check the following parameters to determine if remediation actions are needed:
+  - Cortex XDR alerts related to the hostname by MITRE tactics indicating malicious activity.
```
Comment (umishkin) [inline Packs/Core/Playbooks/playbook-User_added_to_local_administrator_group_using_a_PowerShell_command.yml:12]: Better to use "XSIAM" than "XDR"
An XSIAM customers don't necessarily know XDR

File: Packs/Core/Playbooks/playbook-User_added_to_local_administrator_group_using_a_PowerShell_command.yml [hunk 11]
```diff
@@ -0,0 +1,725 @@
...
+    isoversize: false
+    isautoswitchedtoquietmode: false
+  "6":
+    id: "6"
+    taskid: 22756e65-c2a2-43a1-8192-b98244e84591
+    type: condition
+    task:
+      id: 22756e65-c2a2-43a1-8192-b98244e84591
+      version: -1
+      name: Check for unassigned CGO or OS process
```
Comment (umishkin) [inline Packs/Core/Playbooks/playbook-User_added_to_local_administrator_group_using_a_PowerShell_command.yml:91]: unsigned, not unassigned

**Review by umishkin — APPROVED** [review]

**Review by ssokolovich — APPROVED** [review]

**Pull request merged**